bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/codemining/LineContentCodeMining.java

```diff
@@ -26,6 +26,14 @@
  */
 public abstract class LineContentCodeMining extends AbstractCodeMining {
 
+	/**
+	 * indicates if code mining should be rendered after given position; cursor and selection does
+	 * not include the code mining if set to true.
+	 *
+	 * @since 3.26
+	 */
+	protected boolean afterPosition= false;
```

Comment:
As there is a getter, couldn't this one be `private final` too?

Reply:
I fully agree, thanks for the feedback.